### ISSUE TITLE
Refined Github creates white background on my reaction icons

### ISSUE THREAD

**Issue by coreyfarrell** [StylishThemes/GitHub-Dark#1035]
Refined Github has a style:
```css
.reaction-summary-item.user-has-reacted {
	--background: #f2f8fa;
}
```

This results in an oddly out of place white background for the icon of my own reactions.  I've fixed this in my own browser by adding a custom userstyle to my browser:
```css
/* ==UserStyle==
@name           Refined-Github Dark fix
@namespace      github.com/cfware
@version        1.0.0
@description    Fix conflict refined-github dark conflict
@author         Me
==/UserStyle== */

@-moz-document domain("github.com") {
    .reaction-summary-item.user-has-reacted {
        --background: transparent!important;
    }
}
```

Would Github-Dark consider adding something to like this?

* **Browser**: Firefox 70
* **Operating System**: Fedora 30
* **Link to page with the issue**: https://github.com/facebook/jest/pull/9192#discussion_r348574868
* **Screenshot**:
![Screenshot from 2019-11-20 13-06-55](https://user-images.githubusercontent.com/903597/69265229-daad1e80-0b96-11ea-95df-98efc9b50067.png)

**Comment by silverwind** [comment]
What is this `--background` syntax exactly? I've never seen a variable in a propery name. I assume replacing with `background` works as expected?

**Comment by coreyfarrell** [comment]
`--background` is a [CSS Variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) being set for that specific selector and its children.  Just setting background on this class would not work as the `--background` variable is actually being applied to an `img` within:

```css
.reaction-summary-item img {
	max-width: 100%;
	border-radius: inherit;
	background-color: var(--background);
}
```

Simply overwriting the background of this img does also work.

**Comment by silverwind** [comment]
Ok, odd variable name but if it only applies to a element and children, it's probably okay.

Can you show a screenshot of how these own reactions look with our style disabled?

**Comment by coreyfarrell** [comment]
With Refined-Github but no additional style:
![Screenshot from 2019-11-20 16-01-34](https://user-images.githubusercontent.com/903597/69278023-7480c580-0baf-11ea-8e8c-6533a96d5fe9.png)

With Refined-Github + GitHub-Dark + my added `--background` patch all enabled:
![Screenshot from 2019-11-20 16-01-57](https://user-images.githubusercontent.com/903597/69278082-9417ee00-0baf-11ea-82be-483435bfe815.png)

**Comment by silverwind** [comment]
Ok, seems resonable to add as a manual style. Our automatic generator can not yet pick up variables like this.

**Comment by silverwind** [comment]
Pushed and released as v1.22.142, let us know if it works.

**Comment by coreyfarrell** [comment]
Sorry to have created confusion, you added `--background: #f2f8fa`.  That is the value from Refined Github (nearly white).  It needs to instead be `--background: transparent`.

**Comment by silverwind** [comment]
Oops, fixed and pushed v1.22.143.

**Comment by coreyfarrell** [comment]
Looks great now, thank you!
